docs/contributor/03-60-regions-configuration.md

```diff
@@ -0,0 +1,45 @@
+# Regions and Zones Configuration
+
+Most of the Kyma plans requires a region to be specified in the request body (provisioning parameters). The regions configuration  allows you to control which regions are available for the Kyma plan and a platform region. Additionally, every region has different set of zones which can be sleected.
+
+## Allowed regions
+
+The `plansConfiguration` section of the `values.yaml` file contains a list of plans. Every plan has a platform region with allowed regions. To avoid specifying all possible platform regions you can define `default` values, for example:
+
+```yaml
+
+plansConfiguration:
+  aws:
+    cf-eu11:
+      - eu-central-1
+    default:
+      - us-east-1
+      - us-west-2
+```
+
+The above configuration means that the `cf-eu11` platform region is allowed to use only the `eu-central-1` provider region, while all other platform regions are allowed to use `us-east-1` and `us-west-2` provider regions.
+
+## Display names and zones
+
+The json schema, which defines allowed regions, contains a list of a region display name, which is shown to the user. All region must have corresponding display name and set of zones defined in the `providersConfiguration`. The display name is shown to the user in the UI, while the zones are used by the provisioning process to create a worker node pool.
+```yaml
+providersConfiguration:
+  aws:
+    regions:
+      eu-central-1:
+          displayName: "eu-central-1 (Europe, Frankfurt)"
+          zones: [ "a", "b", "c" ]
+      eu-west-2:
+        displayName: "eu-west-2 (Europe, London)"
+        zones: [ "a", "b", "c" ]
+  azure:
+    regions:
+      brazilsouth: 
+        displayName: "Brazil South"
+        zones: [ "1", "2", "3" ]
+      canadacentral: 
+        displayName: "Canada Central"
+        zones: [ "1", "2", "3" ]
+```
+
+The above configuration defines display names and zones for AWS and Azure regions. You can describe regions for the following providers: azure, aws, gcp, sap-converged-cloud.
```

Comment:
While we can refer to plans using technical names, such as `aws` or `azure,` I believe we should use the full proper names of providers: Microsoft Azure, Amazon Web Services, Google Cloud, and SAP Cloud Infrastructure. WDYT?

Reply:
The line 45 describes possible values (keys in the structure) in the config. It would be hard to write something like:
```
providersConfigurations:
  "Microsoft Azure":
    regions:
      brazilsouth:
        displayName: "Brazil South"
        zones: [ "1", "2", "3" ]
```

Those values described in the line 45 are used in the configuration, not for user interface. On the other hand, the display name is shown to the user.

Reply:
In that case, it's a good idea to use code font for the names you originally used.

You can describe regions for the following providers: `azure`, `aws`, `gcp`, `sap-converged-cloud`.